pkg/deploy/generator/resources_rp.go

```diff
@@ -890,6 +891,17 @@ mkdir -p /usr/lib/ssl/certs
 csplit -f /usr/lib/ssl/certs/cert- -b %03d.pem /etc/pki/tls/certs/ca-bundle.crt /^$/1 {*} >/dev/null
 c_rehash /usr/lib/ssl/certs
 
+cat >/etc/default/vsa-nodescan-agent.config <<EOF
```

Comment:
I'm not sure that I understand: Where do we install the agents itself

Reply:
The binary that runs is the `azsec` binary that is already part of vmss template. I am just adding additional configuration. I've followed instructions (below) that is supposed to auto-"magically" download the Qualys plugin needed for azsec process to perform scan/
![image](https://user-images.githubusercontent.com/4564384/125108023-79a9c580-e0af-11eb-98dc-9d3d86d1b296.png)

Reply:
I am following the recommended method, but I am not exactly sure specific details of how Qualys plugin does get downloaded. I assume that the Managed Identity that is allowlisted reports being online via `azsec` process to central scanning server, and then that process downloads Qualys plugin and is able to perform scans.  

https://microsoft.sharepoint.com/teams/VulnerableNodes/SitePages/Acquiring-the-private-Qualys-plugin.aspx (require msft VM or SAW)